sources/packages/forms/src/form-definitions/studentprofile.json

```diff
@@ -3449,6 +3449,365 @@
+            "style": "",
+            "page": "",
+            "left": "",
+            "top": "",
+            "width": "",
+            "height": ""
+          },
+          "type": "textarea",
+          "wysiwyg": false,
+          "input": true,
+          "refreshOn": "",
+          "dataGridLabel": false,
+          "allowMultipleMasks": false,
+          "addons": [],
+          "mask": false,
+          "inputType": "text",
+          "inputMask": "",
+          "fixedSize": true,
+          "id": "e6gwlmg"
+        },
+        {
+          "label": "I Agree to the terms and conditions of the StudentAid BC Declaration",
+          "labelWidth": "",
+          "labelMargin": "",
+          "description": "",
+          "tooltip": "",
+          "shortcut": "",
+          "inputType": "checkbox",
+          "customClass": "",
+          "tabindex": "",
+          "hidden": false,
+          "hideLabel": false,
+          "autofocus": false,
+          "disabled": false,
+          "tableView": false,
+          "modalEdit": false,
+          "defaultValue": false,
+          "persistent": true,
+          "protected": false,
+          "dbIndex": false,
+          "encrypted": false,
+          "redrawOn": "",
+          "clearOnHide": true,
+          "customDefaultValue": "",
+          "calculateValue": "",
+          "calculateServer": false,
+          "allowCalculateOverride": false,
+          "validate": {
+            "required": true,
+            "customMessage": "",
+            "custom": "",
+            "customPrivate": false,
+            "json": "",
+            "strictDateValidation": false,
+            "multiple": false,
+            "unique": false
+          },
+          "errorLabel": "",
+          "errors": "",
+          "key": "sinConsent",
+          "tags": [],
+          "properties": {},
+          "conditional": {
+            "show": "",
+            "when": "",
+            "eq": "aest-account-approval",
+            "json": ""
+          },
+          "customConditional": "",
+          "logic": [
+            {
+              "name": "Disable checkbox for Client AEST",
+              "trigger": {
+                "type": "javascript",
+                "javascript": "result = (data.mode && data.mode === 'aest-account-approval');"
+              },
+              "actions": [
+                {
+                  "name": "Disable Checkbox",
+                  "type": "property",
+                  "property": {
+                    "label": "Disabled",
+                    "value": "disabled",
+                    "type": "boolean"
+                  },
+                  "state": true
+                }
+              ]
+            }
+          ],
+          "attributes": {},
+          "overlay": {
+            "style": "",
+            "page": "",
+            "left": "",
+            "top": "",
+            "width": "",
+            "height": ""
+          },
+          "type": "checkbox",
```

Comment:
why this checkbox?

Reply:
I will remove it.

Reply:
Actually checking again, it is correct. It was generated under the custom logic to disable/enable the checkbox for SIN consent. You can also check in forms io.